gateway/src/apicast/policy/liquid_context_debug/context_content.lua

```diff
@@ -0,0 +1,97 @@
+local type = type
+local pairs = pairs
+local tostring = tostring
+
+local _M = {}
+
+-- The context usually contains a lot of information. For example, it includes
+-- the whole service configuration. Also, some of the values are objects that
+-- we can't really use when evaluating liquid, like functions.
+-- That's why we define only a few types of keys and values to return.
+local accepted_types_for_keys = {
+  string = true,
+  number = true,
+}
+
+local accepted_types_for_values = {
+  string = true,
+  number = true,
+  table = true
+}
+
+-- In the context, there might be large integers as keys. This can be a problem
+-- when converting to JSON. Imagine an entry of the table like this:
+-- { [1000] = 'something' }. To convert that to JSON, we need an array of 1000
+-- positions with only one occupied. With arrays like these, 'cjson' raises an
+-- "Excessively sparse arrays" error:
+-- https://github.com/efelix/lua-cjson/blob/4f27182acabc435fcc220fdb710ddfaf4e648c86/README#L140
+-- For example, there is a lrucache instance with a table that maps service ids
+-- (numbers) to hosts. So we can have something like { [123456] = "127.0.0.1" }.
+-- This can be limited using a global setting of the 'cjson' library. However,
+-- we're going to implement an add-hoc solution here so we don't affect the
+-- other modules. We're going to convert those large ints to strings as a
+-- workaround.
+local max_integer_key = 10
```

Comment:
I think arrays with 10 elements are common. Maybe better to set this to a something like 1000 ?

Reply:
OK. I'll raise the limit to 1k. I think that in practice large service IDs are going to be the problem.